lib/csl/bibtex_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Csl
+
+  # Convert BibTexIngester content into a CSL format
+  class BibtexMapper
+
+    # Convert BibTexIngester authors into CSL authors
+    # @param [Array<Hash>] authors array of hash data
+    # @return [Array<Hash>] CSL authors array of hash data
+    def self.authors_to_csl(authors)
```

Comment:
To clarify, you are relying on the previously established test coverage on `Csl::Mapper` to exercise these newly public methods?

Reply:
Yes; and they do.  But now that they are public, that seems to entail they should have their own specs.  I'll see if I can quickly add specs (I just don't want to waste too much time on it, as they are sorta tested already).